docs/intro/tutorial01.txt

```diff
@@ -257,10 +257,11 @@ and put the following Python code in it:
     def index(request):
         return HttpResponse("Hello, world. You're at the polls index.")
 
-This is the simplest view possible in Django. To call the view, we need to map
-it to a URL - and for this we need a URLconf.
+To access this view in the browser, we need to map it to a URL - and for this
+we need what we call a URL configuration, or a "URLconf" for short.
```

Comment:
I would keep the first sentence (please wrap at 79cols):
```suggestion
This is the simplest view possible in Django. To access it in a browser, we need to map it to a URL - and for this
we need to define a URL configuration, or "URLconf" for short.
```

Reply:
We try to avoid using words like simple/easy/just and so on. See ed2d411aa84efc76baba3adf0d0f99df0e44ba57

Reply:
@nessita I like your changes better! It's more concise and easier to read. 
I initially removed the "simplest view" sentence because, up until this point, it was never defined what a "View" actually is. But then again, I think that is out of the scope of this tutorial, so we can leave that sentence in.

As for @smithdc1 's comment: though I do agree with that rule and think it makes a lot of sense, does it apply to this case? We're talking about the view itself and not any tasks or operations that could refer to the user's skill level.